serde_v8/src/magic/value.rs

```diff
@@ -29,50 +27,22 @@ impl<'s> From<Value<'s>> for v8::Local<'s, v8::Value> {
   }
 }
 
-impl serde::Serialize for Value<'_> {
-  fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
-  where
-    S: serde::Serializer,
-  {
-    use serde::ser::SerializeStruct;
-
-    let mut s = serializer.serialize_struct(NAME, 1)?;
-    let mv = Value {
-      v8_value: self.v8_value,
-    };
-    let hack: u64 = unsafe { std::mem::transmute(mv) };
-    s.serialize_field(FIELD, &hack)?;
-    s.end()
+impl ToV8 for Value<'_> {
+  fn to_v8<'a>(
+    &self,
+    _scope: &mut v8::HandleScope<'a>,
+  ) -> Result<v8::Local<'a, v8::Value>, crate::Error> {
+    // SAFETY: not fully safe, since lifetimes are detached from original scope
```

Comment:
Then this function should be unsafe to make the caller aware of this.

Reply:
This is never directly called by end-users. It's the result of attempting to de/serialize `serde_v8::Value`s